models/containerized_model/README.md

```diff
@@ -185,3 +185,56 @@ This should create a docker container and host the webservice.
 
 ### Step 6 - Test
 Make a request to `http://127.0.0.1:8551/predict` with the respective parameters.
+
+
+## [Proxy Template](#proxy-template)
+### Step 1 - Template generation
+
+Generate the code template for containerization of your model:
+```
+./generate.sh -d generated-container-proxy -i certifai-proxy-container:latest -m proxy -n containerproxy -r container-proxy
+```
+
+This command should create a directory called `generated-container-proxy`
+in your current directory with the generated code.
+
+For more `generate` options:
+```
+./generate.sh --help
+```
+
+### Step 2 - Copy artifacts
+Copy the `packages` folder from inside the toolkit into the generated
+directory `generated-container-proxy`:
+
+```
+cp -r <certifai-toolkit-path>/packages generated-container-proxy/packages
+```
```

Comment:
There's a step missing here, where they need to edit the transform_request_to_hosted_model_schema and transform_response_to_certifai_predict_schema methods.

Reply:
added